### PR TITLE
simplified alphabet annotation macro definition

### PR DESCRIPTION
**Pull request by jnegrea**
I made macros to automatically generate annotation macros for lists of characters or latex macros (for greek letters)... 
I implemented all existing annotations in this way. 
A couple annotation types had too many exceptions so I changed the annotation prefix so there would be fewer exceptions.
I think this should be abstracted further because i) there is a combinatorial explosion of possible annotation combinations and ii) this defines thousands of macros, most of which will not be used, and im not sure whether this is ideal or affects latex performance

### CODE REVIEW

File: shortex.sty [hunk 1]
```diff
@@ -476,773 +476,99 @@
 %%% Symbol shortcuts %%%
 %%%%%%%%%%%%%%%%%%%%%%%%
 
-% Math bold italics
-\def\mathbbi#1{\textbf{\em #1}}
 
+\newcommand{\charalphabetmacro}[3]{
```
Comment (jhuggins) [inline shortex.sty:480]: Could you add a comment describing what this does?

File: shortex.sty [hunk 2]
```diff
@@ -476,773 +476,99 @@
...
+    \def\mydeffoo##1{\expandafter\def\csname #1##1\endcsname{#2{##1}}}
+    \def\mydefallfoo##1{\ifx##1\mydefallfoo\else\mydeffoo##1\expandafter\mydefallfoo\fi}
+    \expandafter \mydefallfoo #3\mydefallfoo
+}
+
+\newcommand{\texalphabetmacro}[3]{
```
Comment (jhuggins) [inline shortex.sty:486]: Comment here too

File: example-jn.tex [hunk 3]
```diff
@@ -0,0 +1,42 @@
+\documentclass{article}
```
Comment (jhuggins) [inline example-jn.tex:1]: Could you add these to `example.tex`?

**Review by jhuggins — CHANGES_REQUESTED** [review]
Nice! You'll need to pull the latest changes from `master`? I just merged my PR.

**Comment by jnegrea** [comment]
i pushed resolutions to @jhuggins requests to the branch, do i need to merge the pull request to update it? or does that push it to master?

**Review by jhuggins — APPROVED** [review]
Nice! I think this is fine to merge after deleting `example-jn.*`

**Comment by jhuggins** [comment]
> i pushed resolutions to @jhuggins requests to the branch, do i need to merge the pull request to update it? or does that push it to master?

Yup. Merge the pull request, then you can delete your branch.

**Pull request merged**